pkg/volume/util/util.go

```diff
@@ -545,7 +545,7 @@ func GetPluginMountDir(host volume.VolumeHost, name string) string {
 // IsLocalEphemeralVolume determines whether the argument is a local ephemeral
 // volume vs. some other type
 func IsLocalEphemeralVolume(volume v1.Volume) bool {
-	return volume.GitRepo != nil ||
+	return volume.Secret != nil || volume.GitRepo != nil ||
```

Comment:
I think If  secret should be included, configmap should also be included

Reply:
configmap is already checked